libcontainer/container_linux.go

```diff
@@ -1299,7 +1299,7 @@ func (c *linuxContainer) Restore(process *Process, criuOpts *CriuOpts) error {
 	if err != nil {
 		return err
 	}
-	defer unix.Unmount(root, unix.MNT_DETACH)
+	defer func() { _ = unix.Unmount(root, unix.MNT_DETACH) }()
```

Comment:
For the `go funcWhichReturnsErr()` cases I would go with `// nolint` because the alternative is quite a bit uglier (and I think in some cases may have strange behaviour with variable values -- because IIRC closures capture variables slightly differently to `go` statements).

Reply:
I was thinking about suggesting something like that, too. Having to add a whole function call to satisfy a linter is borderline overkill.

Reply:
changed those to use `//nolint: errcheck`